CMakeLists.txt

```diff
@@ -13,8 +13,12 @@
 cmake_minimum_required(VERSION 2.8.4 FATAL_ERROR)
 cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}.${CMAKE_PATCH_VERSION})
 
+IF (NOT DEFINED KFS_DIR_PREFIX)
+    set (KFS_DIR_PREFIX ${CMAKE_SOURCE_DIR})
```

Comment:
I mean that QFS is now an external project to that other source code you are talking about. They no longer live in the same source tree and so we should use solutions for that situation. That is a solved problem. The internal project you are talking about should add qfs as an external project through cmake, link against it's shared or static libraries, and call any APIs that it needs in that way. It is probably doing just that right now but expecting some filesystem path structure. That's what I want to move away from.

How about this for now: remove this change from this pull request and make another pull request for this change specifically. That way, we can go forward with this bug fix while we still discuss this change. What do you think?